rules/integrations/cloud_defend/lateral_movement_ssh_process_launched_inside_a_container.toml

```diff
@@ -0,0 +1,72 @@
+[metadata]
+creation_date = "2023/05/12"
+integration = ["cloud_defend"]
+maturity = "production"
+min_stack_comments = "New Integration: Cloud Defend"
+min_stack_version = "8.8.0"
+updated_date = "2023/05/12"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects an SSH or SSHD process executed from inside a container. This includes both the client ssh binary and server ssh daemon process. 
+SSH usage inside a container should be avoided and monitored closely when necessary. With valid credentials an attacker may move laterally to 
+other containers or to the underlying host through container breakout. They may also use valid SSH credentials as a persistence mechanism.
+"""
+false_positives = ["""
+  SSH usage may be legitimate depending on the environment. 
+  Access patterns and follow-on activity should be analyzed to distinguish between authorized and potentially malicious behavior.
+  """
+]
+from = "now-6m"
+index = ["logs-cloud_defend*"]
+interval = "5m"
+language = "eql"
+license = "Elastic License v2"
+name = "SSH Process Launched From Inside A Container"
+references = [ 
+    "https://microsoft.github.io/Threat-Matrix-for-Kubernetes/techniques/SSH%20server%20running%20inside%20container/",
+    "https://www.blackhillsinfosec.com/sshazam-hide-your-c2-inside-of-ssh/",
+]
+risk_score = 73
+rule_id = "03a514d9-500e-443e-b6a9-72718c548f6c"
+severity = "high"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Lateral Movement", "Persistence", "Container"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where container.id: "*" and event.type== "start" and
```

Comment:
Why filter by container.id * if it's all container ids?

Reply:
This is the field we're using to keep these rules from triggering on normal linux hosts or VMs, similar to the addition of `host.os` to detection rules